symfony

```diff
@@ -0,0 +1,188 @@
+#!/bin/sh
```

Comment:
It's a bash executable which is some kind of an alias for basic commands without you even need to know the container name. Just run `./symfony` from the root directory and the information how to use this will be show. 

```
usage : ./symfony stats|console|composer|enter

  stats      Displays live resource usage statistics of all containers
  console    Executes app/console in the symfony root directory
  composer   Executes composer in the symfony root directory
  enter      Enters the bash of a given container type (e.g. php, nginx)
```

`./symfony stats` will show usefull information about the containers
`./symfony console` is an alias on your host for symfony `app/console` (or `bin/console`)
`./symfony composer` is an alias on your host for `composer`
`./symfony enter [servicename]` will have you open a bash shell on the specified container

They are just some usefull shorthand tools, without you having to type: `docker-compose exec [service] [command] [parameters]` or `docker exec -it [container] [command] [parameters]`